emails/models.py

```diff
@@ -137,36 +184,35 @@ def __str__(self):
     def delete(self, *args, **kwargs):
         # TODO: create hard bounce receipt rule in AWS for the address
         deleted_address = DeletedAddress.objects.create(
-            address_hash=sha256(self.address.encode('utf-8')).hexdigest(),
+            address_hash=address_hash(self.address),
             num_forwarded=self.num_forwarded,
             num_blocked=self.num_blocked,
             num_spam=self.num_spam,
         )
         deleted_address.save()
         profile = Profile.objects.get(user=self.user)
-        profile.address_last_deleted = datetime.now()
+        profile.address_last_deleted = datetime.now(timezone.utc)
         profile.num_address_deleted += 1
         profile.save()
         return super(RelayAddress, self).delete(*args, **kwargs)
 
-    def make_relay_address(user, num_tries=0):
+    def make_relay_address(user_profile, num_tries=0):
```

Comment:
Changed the param here to use `user_profile` instead because I want to ensure that the locked_profile passed from views/emails.py is used. I can also change the `make_domain_address` to use `user_profile` instead, wdyt?

Reply:
d2577fb90fe1cb906a442801fb114c5a94b024f4